### ISSUE TITLE
IE11: Gallery captions are positioned incorrectly. 

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#8594]
**Describe the bug**

On IE11, gallery captions appear below the image, rather than on top of it. They're also hidden by the "Upload an image" drop zone. In the deselected state, the captions are pushed out far to the right of where they're supposed to be. 

**To Reproduce**

1. Use IE11.
2. Add a Gallery block.
3. Attempt to add a caption.

**Animated GIF**

![gallery-captions-ie11](https://user-images.githubusercontent.com/1202812/43719936-236a597e-995d-11e8-85b7-9306206f02ea.gif)

**Tested in:**
Windows 7, IE 11.0.9600.17420
Gutenberg 3.4

**Comment by webmandesign** [comment]
This could be fixed by applying actual `left/right` and `bottom` position for the [`figcaption` absolute positioning styles](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/gallery/style.scss#L32). So, basically adding this:
```css
figcaption {
	position: absolute;
	left: 0;
	bottom: 0;
	...
```

I didn't go ahead and created a pull request fixing this as I'm not really sure why there is this [flebox set for the gallery `figure`](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/gallery/style.scss#L21)... So, can anybody explain it? What's the purpose of it? What was the desired layout?

If the flexbox from `figure` is removed, you can even omit the `left: 0;` styles above.

**Comment by jasmussen** [comment]
This PR is fixed by #9762.

Thanks again for the advice here.

**Comment by kjellr** [comment]
Closed via #9762.
